jbpm-wb-showcase/pom.xml

```diff
@@ -392,6 +392,10 @@
           <artifactId>slf4j-jdk14</artifactId>
           <groupId>org.slf4j</groupId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.geronimo.specs</groupId>
+          <artifactId>geronimo-activation_1.1_spec</artifactId>
+        </exclusion>
```

Comment:
This should not be (hopefully) needed after we merge https://github.com/droolsjbpm/jbpm-designer/pull/556